src/exactlyOnceTest/java/com/clickhouse/ExactlyOnceTest.java

```diff
@@ -138,24 +141,47 @@ public static void tearDown() throws IOException, URISyntaxException, Interrupte
         LOGGER.info(String.valueOf(confluentAPI.deleteTopic("test_exactlyOnce_offsets_" + topicCode)));
         LOGGER.info(String.valueOf(confluentAPI.deleteTopic("test_exactlyOnce_status_" + topicCode)));
         LOGGER.info(String.valueOf(confluentAPI.deleteTopic("test_exactlyOnce_data_" + topicCode)));
+        LOGGER.info(String.valueOf(confluentAPI.deleteTopic("test_exactlyOnce_data_multi_" + topicCode)));
         LOGGER.info(String.valueOf(clickhouseAPI.dropTable("test_exactlyOnce_data_" + topicCode)));
+        LOGGER.info(String.valueOf(clickhouseAPI.dropTable("test_exactlyOnce_data_multi_" + topicCode)));
     }
 
+
     @Test
-    @Order(1)
-    public void basicTest() throws InterruptedException, ExecutionException, TimeoutException, URISyntaxException, IOException {
-        assertEquals(1, 1);
-        clickhouseAPI.getServiceState(properties.getProperty("clickhouse.cloud.serviceId"));
+    public void checkTotalsEqual() throws InterruptedException, ExecutionException, TimeoutException {
+        String topicName = "test_exactlyOnce_data_" + topicCode;
+
+        ClickHouseResponse clickHouseResponse = clickhouseAPI.clearTable(topicName);
+        LOGGER.info(String.valueOf(clickHouseResponse));
+        clickHouseResponse.close();
+
+        Integer count = sendDataToTopic(topicName);
+
+        Thread.sleep(60000);
```

Comment:
I would inspect the topic offset instead of doing fixed sleep. (Same as I did in the integration test)